operatorcourier/validate.py

```diff
@@ -273,7 +273,10 @@ def _csv_spec_validation(self, spec, bundleData):
                     if 'version' in csvOwnedCrd:
                         if 'spec' in crd:
                             if 'versions' in crd['spec']:
-                                if csvOwnedCrd['version'] not in crd['spec']['versions']:
+                                if csvOwnedCrd['version'] not in [
+                                    v['name'] for v in crd['spec']['versions']
+                                    if 'name' in v
+                                ]:
                                     self._log_error('CSV.spec.crd.owned.version is'
```

Comment:
Just a trivial fix to a minor thing - when this error spits out - there should be a space between "is" and "not" but instead it prints out as "... version isnot in CRD..."

```suggestion
                                    self._log_error('CSV.spec.crd.owned.version is '
```

Reply:
Done, thanks.